knownprojects_build/python/upload_dob_match_to_carto.py

```diff
@@ -0,0 +1,17 @@
+import os
+from helper.engines import recipe_engine, edm_engine, build_engine
+from cartoframes.auth import set_default_credentials
+from cartoframes import to_carto
+from shapely import wkb
+import geopandas as gpd
+
+year = 'test'
+
+set_default_credentials(
+    username=os.environ.get('CARTO_USERNAME'),
+    api_key=os.environ.get('CARTO_APIKEY')
+)
+
+sql = 'select * from dob_review'
```

Comment:
I would use a similar schema as the first round cluster review. It excluded `adjusted_units` as well as the `proportion_built_by_` columns